src/marked.js

```diff
@@ -204,7 +258,12 @@ marked.walkTokens = function(tokens, callback) {
         break;
       }
       default: {
-        if (token.tokens) {
+        if (marked.defaults?.extensions?.walkableTokens?.[token.type]) { // Walk any extensions
+          marked.defaults?.extensions.walkableTokens[token.type].forEach(function(walkableTokens) {
+            marked.walkTokens(token[walkableTokens], callback);
+          });
+        }
+        if (token.tokens && !marked.defaults?.extensions?.walkableTokens[token.type]?.tokens) {
```

Comment:
Should we still walk `tokens` if it isn't in `walkableTokens`? Could that property mean something else for that token or should we make that property always walkable?

Reply:
My instinct was always walk `tokens`. We can say so in the documentation, and then users don't have to bother with making a walkabletokens if their extension is simple enough.

Reply:
ya I think we should walk `tokens` if they don't provide `walkableTokens` but if they do create `walkableTokens` I think it would be more intuitive if we just walk those properties even if they have a `tokens` property that isn't in `walkableTokens`. If they have `walkableTokens` and they don't add `tokens` I would guess it would be because they don't want it walked.

Reply:
Can we rename `walkableTokens` to `childTokens` or something like that?

Reply:
Yeah we can do that.

Reply:
Ok renamed to childTokens and updated a unit test to show `token.tokens` is only walked by default if `childTokens` isn't provided.